Bangkok Post.js

```diff
@@ -0,0 +1,175 @@
+{
+	"translatorID": "7f74d823-d2ba-481c-b717-8b12c90ed874",
+	"label": "Bangkok Post",
+	"creator": "Matt Mayer",
+	"target": "https://www.bangkokpost.com/.*",
```

Comment:
1) `^` at the beginning to restrict to the beginning of the URL

2) Either this or detectWeb() needs to be more restrictive to keep it from returning `newspaperArticle` for all pages on the site (e.g., https://www.bangkokpost.com/subscription). It looks like the target could potentially be limited to a URL with a `[0-9]+` as the second or third segment in the path, or you could check for something unique to articles in the page DOM from detectWeb().

Reply:
Thank you so much for the detailed feedback! What's the correct behavior in detectWeb if I don't want to handle a URL? Do I return undefined?

Reply:
Return false.

Reply:
I made the regex stricter so it looks for paths like /foo/bar/1234567 or /foo/1234567 as you suggested